packages/block/test/util.ts

```diff
@@ -14,8 +14,12 @@ function createBlock(
   uncles?: BlockHeader[],
   common?: Common
 ): Block {
-  uncles = uncles ?? []
   common = common ?? new Common({ chain: Chain.Mainnet })
+  if (common.isActivatedEIP(3675)) {
+    uncles = []
```

Comment:
nice, i'm thinking we can also do this in `block.ts` under each `// parse uncle headers` section, after `const uncleHeaders = []` wrap the `for (const uncleHeaderData of uhsData` loop in a `if (!common.isActivatedEIP(3675)) {` so if activated it will skip populating the array.

Reply:
Yes, will also remove this for now (tests are actually also passing without this).

In the block constructor if uncleHeaders are provided we should rather also throw.